debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -907,4 +920,54 @@ protected Object convertString(Column column, Field fieldDefn, Object data) {
         }
         return super.convertString(column, fieldDefn, data);
     }
+
+    /**
```

Comment:
Let's avoid such empty comments. But could you add one to the method in the base class, describing its purpose?

Reply:
By moving associating the right jdbc type and native type to a column at construction time, these `Types.DISTINCT` specific handler methods were removed as they're no longer needed.